kubernetes/kubernetes-client-core.cabal

```diff
@@ -51,11 +51,11 @@ library
     , iso8601-time >=0.1.3 && <0.2.0
     , microlens >= 0.4.3 && <0.5
     , mtl >=2.2.1
-    , network >=2.6.2 && <2.9
+    , network >=2.6.2 && <3.9
```

Comment:
I had intentionally chosen a very lax bound on network because I didn't think that any of the 3.X changes would be breaking for us, but if you're concerned about it I could make it lower in a subsequent PR

Reply:
No, I just wanted to make sure that this was intentional. If we face problems, we can always change it.